packages/storage-plus/src/de.rs

```diff
@@ -221,11 +164,6 @@ impl<T: KeyDeserialize, U: KeyDeserialize, V: KeyDeserialize> KeyDeserialize for
 
         Ok((T::from_slice(t)?, U::from_slice(u)?, V::from_slice(v)?))
```

Comment:
Same as the (T, U) tuples (from_vec)

Reply:
Did it. It's a little unclear with the var names. And, not sure it's more efficient.

If we want a self-contained version, that is, one that doesn't depend on `from_slice`, we can always use `from_vec(t.to_vec())`

Reply:
In fact, for triples, the version with `split_off()` has four vec allocations instead of three; because of the small vec for the length of `V. Plus, it requieres a conversion to `to_slice()` for calling `try_into()`.

I really like the previous version better.